opencti-platform/opencti-graphql/src/migrations/1704893567321-capability-publish-knowledge.js

```diff
@@ -1,16 +0,0 @@
-import { executionContext, SYSTEM_USER } from '../utils/access';
-import { addCapability } from '../domain/grant';
-
-export const up = async (next) => {
-  const context = executionContext('migration');
-  await addCapability(context, SYSTEM_USER, {
```

Comment:
I don't think we should remove existing migrations, we can only add newer ones, but we shouldn't delete the existing ones. no ?

Reply:
this migration  had not been played and has not been removed, it has been merged with another (`investigations-and-dashboards-capabilities` ) in order to group together dashboard migrations.

Reply:
not sure to understand how the migration has not been played, since it has been merged in master two months ago and has been part of 6.0 release https://github.com/OpenCTI-Platform/opencti/commit/f266553f69206a4e26c9c3a2e8fdc32373412651#diff-7232600645a940725cb4961bf097b9b14fcc855af0e627b94a3a037e47a5785d

Reply:
we forgot to change the timestamp before merging

Reply:
Ok I understand now that the migration was merged with an older timestamp than the last one (that has even been released) so there are many platforms that haven't run that migration

Reply:
done